dex/src/instruction.rs

```diff
@@ -99,6 +100,36 @@ pub struct SendTakeInstruction {
     pub limit: u16,
 }
 
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
+#[cfg_attr(test, derive(Arbitrary))]
+pub struct NewOrderInstructionV4 {
+    pub side: Side,
+
```

Comment:
feels like these spaces are unnecessary? also does the `cfg_attr` do anything for us?

Reply:
copied from the same formatting as the other instructions, i am indifferent but can remove all the spaces and the cfg_attr its probably test only

Reply:
cool